jpegenc/models/rgb2ycbcr/commons.py

```diff
@@ -0,0 +1,29 @@
+#!/bin/python
```

Comment:
Can i user enable.next = 1 or i have to use enable.next = True. What is more correct?